cmd/cksum.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * cmd/cksum.h
+ * Purpose: calculate and print POSIX checksums for a specific string.
+ *
+ *  This tool is based on cksum tool from Linux coreutils.
```

Comment:
If the code is derived from GNU coreutils, which is what I assume is meant by “Linux coreutils,” then this code is actually GPL and not ISC licensed.

Reply:
And what do you need to be correct from point of view of license?